tests/NuGet.IndexingTests/TestSupport/Constants.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Lucene.Net.Search;
+using NuGet.Indexing;
+
+namespace NuGet.IndexingTests.TestSupport
```

Comment:
>TestSupport [](start = 30, length = 11)

nit: No need to have many namespaces, they just add unnecessary usings. Can we leave this in NuGet.IndexingTests?